### PR TITLE
Enable keep-going knob for database restore

### PR DESCRIPTION
**Pull request by MikhailBurdukov**

### CODE REVIEW

File: ch_backup/ch_backup.py [hunk 1]
```diff
@@ -248,9 +248,9 @@ def restore(
                     "Cloud storage source bucket must be set if backup has data on S3 disks"
                 )
 
-        self._context.config["override_replica_name"] = (
-            override_replica_name or self._context.config.get("override_replica_name")
-        )
+        self._context.config[
```
Comment (MikhailBurdukov) [inline ch_backup/ch_backup.py:251]: Unrelated `black` changes

File: ch_backup/cli.py [hunk 2]
```diff
@@ -555,9 +555,9 @@ def restore_command(
     specified_tables: typing.List[TableMetadata] = _key_values_to_tables_metadata(
         tables
     )
-    specified_exclude_tables: typing.List[TableMetadata] = (
-        _key_values_to_tables_metadata(exclude_tables)
-    )
+    specified_exclude_tables: typing.List[
+        TableMetadata
```
Comment (MikhailBurdukov) [inline ch_backup/cli.py:559]: Unrelated black changes

**Review by Alex-Burmak — APPROVED** [review]

**Review by aalexfvk — APPROVED** [review]
